### PR TITLE
Add test case 7.5 for MEASUREMENTS.

### PR DESCRIPTION
**Pull request by Xiaohanjlll**
fix: #117 
fix: #112

### CODE REVIEW

File: doc/7.Measurements.md [hunk 1]
```diff
@@ -277,6 +277,8 @@ Assertion 7.5.2:
 Assertion 7.5.3:
     if (SpdmMeasurementBlock[i].Measurement.DMTFSpecMeasurementValueType.bit_7 = 0) then (SpdmMeasurementBlock[i].Measurement.DMTFSpecMeasurementValueSize == HashSize)
 
+Assertion 7.5.4:
+    When DMTFSpecMeasurementValueType[6:0]=0x5 , DMTFSpecMeasurementValueType[7] shall be set to 1b .
```
Comment (jyao1) [inline doc/7.Measurements.md:281]: 1) Please leave a blank line. Follow the same style.

2) Please use the same style for description, such as AAA == BBB, if CCC, or if () then (AAA == BBB)

**Comment by jyao1** [comment]
please show the test result.

**Comment by Xiaohanjlll** [comment]
> please show the test result.


image

File: library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c [hunk 2]
```diff
@@ -1831,6 +1998,11 @@ common_test_case_t m_spdm_test_group_measurements[] = {
      spdm_test_case_measurements_invalid_request,
      spdm_test_case_measurements_setup_version_any,
      spdm_test_case_common_teardown},
+    {SPDM_RESPONDER_TEST_CASE_MEASUREMENTS_SPDM_MEASUREMENT_BLOCK,
+     "spdm_test_case_measurements_success_measurements_block",
+     spdm_test_case_measurements_success_measurement_block,
+     spdm_test_case_measurements_setup_measurement_block,
```
Comment (jyao1) [inline library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c:2004]: why not use spdm_test_case_measurements_setup_version_any?

File: library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c [hunk 3]
```diff
@@ -1681,6 +1694,160 @@ void spdm_test_case_measurements_invalid_request (void *test_context)
...
+            return;
+        }
+
+        if ((measurement_block->measurement_block_dmtf_header.dmtf_spec_measurement_value_type &
+             SPDM_MEASUREMENT_BLOCK_MEASUREMENT_TYPE_RAW_BIT_STREAM) == 0)
+        {
+            hash_size = libspdm_get_measurement_hash_size(
+                ((libspdm_context_t *)spdm_context)->connection_info.algorithm.measurement_hash_algo);
+
+            /* Assertion 7.5.2: if (SpdmMeasurementBlock[i].Measurement.DMTFSpecMeasurementValueType.bit_7 = 0) then (SpdmMeasurementBlock[i].MeasurementSize == 3 + HashSize) */
```
Comment (jyao1) [inline library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c:1776]: dont exceed 100

File: library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c [hunk 4]
```diff
@@ -1681,6 +1694,160 @@ void spdm_test_case_measurements_invalid_request (void *test_context)
...
+            if (measurement_block->measurement_block_common_header.measurement_size == (hash_size
+                                                                                        + 1 /* DMTFSpecMeasurementValueType */
+                                                                                        + 2 /* DMTFSpecMeasurementValueSize */))
```
Comment (jyao1) [inline library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c:1779]: why not use sizeof(spdm_measurement_block_dmtf_header_t) ?

File: library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c [hunk 5]
```diff
@@ -1681,6 +1694,160 @@ void spdm_test_case_measurements_invalid_request (void *test_context)
...
+                SPDM_RESPONDER_TEST_GROUP_MEASUREMENTS,
+                SPDM_RESPONDER_TEST_CASE_MEASUREMENTS_SPDM_MEASUREMENT_BLOCK, 2,
+                test_result, "SpdmMeasurementBlock MeasurementSize - 0x%02x",
+                measurement_block->measurement_block_common_header.measurement_size);
+            if (test_result == COMMON_TEST_RESULT_FAIL)
+            {
+                return;
+            }
+
+            /* Assertion 7.5.3: if (SpdmMeasurementBlock[i].Measurement.DMTFSpecMeasurementValueType.bit_7 = 0) then (SpdmMeasurementBlock[i].Measurement.DMTFSpecMeasurementValueSize == HashSize) */
```
Comment (jyao1) [inline library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c:1795]: dont exceed 100

File: library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c [hunk 6]
```diff
@@ -1681,6 +1694,160 @@ void spdm_test_case_measurements_invalid_request (void *test_context)
...
+                measurement_block->measurement_block_dmtf_header.dmtf_spec_measurement_value_size);
+            if (test_result == COMMON_TEST_RESULT_FAIL)
+            {
+                return;
+            }
+        }
+
+        if (version >= SPDM_MESSAGE_VERSION_12)
+        {
+            /* Assertion 7.5.4: if (SpdmMeasurementBlock[i].Measurement.DMTFSpecMeasurementValueType.bit_[6:0] == 0x5) then (SpdmMeasurementBlock[i].Measurement.DMTFSpecMeasurementValueType.bit_7 == 1) */
```
Comment (jyao1) [inline library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c:1816]: dont exceed 100

File: library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c (hunk 2 above)
Comment (Xiaohanjlll) [reply library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c]: Thanks. I modified these questions

**Pull request merged**